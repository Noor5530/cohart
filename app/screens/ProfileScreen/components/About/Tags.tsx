import React from "react";
import { View, Text, StyleSheet } from "react-native";
import { iconColor } from "config/colors";
import { widthPercentageToDP } from "react-native-responsive-screen";

export default function Tags(props: { tag: string }) {
  return (
    <View style={styles.container}>
      <Text style={styles.tag}>{props.tag}</Text>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    marginVertical: 5,
    borderWidth: 1,
    borderColor: iconColor,
    borderRadius: 25,
    paddingHorizontal: 10,
    marginHorizontal: 5,
    alignItems: "center",
    justifyContent: "center",
    padding: 10,
  },
  tag: {
    fontSize: widthPercentageToDP(2.8),
    fontWeight: "400",
    color: iconColor,
    textAlign: "center",
  },
});
