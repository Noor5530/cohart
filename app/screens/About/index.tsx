import StatusBar from 'components/CustomStatusBar';
import Header from 'components/Header';
import HeaderHeading from 'components/HeaderHeading';
import { CohartIcon } from 'components/Icons';
import { appFonts } from 'components/text';
import { Container } from 'components/workAround';
import { isIphone } from 'lib/isIphone';
import React from 'react';
import {  ScrollView, Text, View } from 'react-native';
import DeviceInfo from 'react-native-device-info';
import {AboutIcon} from 'components/Icons'
import { iconColor, textColor } from '../../config/colors';
import styles from './style';
import { widthPercentageToDP } from 'react-native-responsive-screen';

const VERSION = DeviceInfo.getVersion();
const BUILD = DeviceInfo.getBuildNumber();

const About = () => {
    return (
      <Container>
        <View key="about" style={{ flex: 1, backgroundColor: textColor }}>
          <StatusBar backgroundColor={textColor} barStyle="dark-content" />
          <Header
            style={{
              paddingTop: isIphone(),
              borderBottomColor: iconColor,
              borderBottomWidth: 1.5,
            }}
            textStyle={{
              color: iconColor,
              fontFamily: appFonts.AktivGroteskEx_Trial_Bd,
            }}
            back
            color={iconColor}
          />
          <HeaderHeading
            title={"About"}
            renderIcon={
              <View style={styles.icon}>
                <AboutIcon
                  width={widthPercentageToDP("15")}
                  height={widthPercentageToDP("15")}
                />
              </View>
            }
            drawer={false}
          />
          <ScrollView
            showsVerticalScrollIndicator={false}
            contentContainerStyle={styles.scroll}
          >
            <CohartIcon />
            <Text style={styles.heading}>
              Connecting the Creative{"\n"}Community
            </Text>
            <Text style={styles.versionInfo}>
              VERSION {VERSION} ({BUILD})
            </Text>
            <Text style={styles.description}>
              Cohart is a female-founded platform which enables visual creators
              and their audiences to build authentic connections and
              communities.{"\n\n"}
              Our focus is on unlocking new ways for visual creators to monetize
              their passions, while building an experience for audiences to
              interact with these creators in a fun and engaging way.{"\n\n"}
              Cohart is for for everyone. We make art discovery accessible by
              opening windows into the worlds of creators.
            </Text>
          </ScrollView>
        </View>
      </Container>
    );
};
export default React.memo(About);
