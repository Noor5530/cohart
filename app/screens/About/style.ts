import { appFonts } from 'components/text';
import { iconColor } from 'config/colors';
import { StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  heading: {
    fontSize: 20,
    fontFamily: appFonts.InterRegular,
    color: iconColor,
  },
  center: {
    alignSelf: "center",
  },
  logo: {
    alignSelf: "center",
    marginTop: 40,
  },
  description: {
    marginTop: 22,
    fontSize: 15,
    fontFamily: appFonts.InterRegular,
    color: iconColor,
  },
  versionInfo: {
    paddingTop: 30,
    fontSize: 14,
  },
  paintImage: {
    marginTop: 10,
    alignSelf: "center",
  },
  aboutImage: {
    height: 450,
    alignSelf: "center",
  },
  bottomImage: {
    height: 450,
    alignSelf: "center",
  },
  contactUs: {
    marginTop: 10,
    fontSize: 13,
    fontFamily: appFonts.InterRegular,
    textAlign: "center",
    color: iconColor,
  },
  email: {
    padding: 10,
    paddingTop: 0,
    fontSize: 18,
    fontFamily: appFonts.InterRegular,
    textAlign: "center",
    fontWeight: "500",
    color: iconColor,
  },
  icon: { marginTop: 5, marginLeft: -25, marginBottom: 4 },
  scroll:{ paddingVertical: 35, paddingHorizontal: 36 }
});

export default styles;
