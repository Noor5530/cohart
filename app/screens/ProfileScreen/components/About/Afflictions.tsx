import React, { FC, useState, useEffect } from "react";
import { View, Text, StyleSheet } from "react-native";
import { appFonts } from "../../../../components/text";
import { widthPercentageToDP } from "react-native-responsive-screen";
import { iconColor } from "../../../../config/colors";
import { TagProps } from "../../types";
import { TouchableOpacity } from "react-native-gesture-handler";
import AntDesign from "react-native-vector-icons/AntDesign";
export const UserTags: FC<TagProps> = (props: TagProps) => {
  const { data = [] } = props;
  const [showMoreButton, toggleShowMoreButton] = useState(false);
  useEffect(() => {
    toggleShowMoreButton(false);
  }, [data.length]);
  return (
    <>
      <View style={styles.container}>
        <View style={styles.line} />
        <View {...props} style={styles.innerContainer}>
          {!showMoreButton
            ? data.slice(0, 7).map((place, index) => {
                return (
                  <View key={index} style={styles.container}>
                    <View style={styles.tagContainer} />
                    <Text style={styles.tag}>{place.trim()}</Text>
                  </View>
                );
              })
            : data.map((place, index) => {
                return (
                  <View key={index} style={styles.container}>
                    <View style={styles.tagContainer} />
                    <Text style={styles.tag}>{place.trim()}</Text>
                  </View>
                );
              })}
        </View>
      </View>
      {data?.length > 7 && (
        <TouchableOpacity
          onPress={() => {
            toggleShowMoreButton((prv) => !prv);
          }}
          style={styles.button}
        >
          <Text style={styles.buttonText}>
            {!showMoreButton ? "SHOW MORE" : "HIDE MORE"}
          </Text>
          <AntDesign
            name={showMoreButton ? "up" : "down"}
            size={12}
            color={"#0000FF"}
          ></AntDesign>
        </TouchableOpacity>
      )}
    </>
  );
};

const styles = StyleSheet.create({
  container: { flexDirection: "row" },
  line: {
    top: 0,
    width: 20,
    left: 0,
    backgroundColor: "white",
    zIndex: 5,
  },
  innerContainer: {
    flexWrap: "wrap",
    flexDirection: "row",
    marginLeft: -15,
    backgroundColor: "white",
    paddingRight: 15,
  },
  tagContainer: {
    height: 4,
    width: 4,
    borderRadius: 2,
    backgroundColor: iconColor,
    alignSelf: "center",
  },
  tag: {
    color: iconColor,
    paddingHorizontal: 10,
    paddingTop: 3,
    fontFamily: appFonts.InterRegular,
    fontSize: widthPercentageToDP(3.5),
    fontWeight: "400",
  },
  button: {
    paddingLeft: 20,
    paddingTop: 10,
    flexDirection: "row",
  },
  buttonText: { color: "#0000FF", fontSize: 10 },
});
