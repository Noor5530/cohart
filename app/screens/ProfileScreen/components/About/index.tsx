import React from "react";
import { View, Text, TouchableOpacity, StyleSheet } from "react-native";
import ProfileImages from "../ProfileImages";
import { widthPercentageToDP } from "react-native-responsive-screen";
import { iconColor } from "config/colors";
// import Tags from "./Tags";
import moment from "moment";
import { appFonts } from "../../../../components/text";
import { AboutProps } from "screens/ProfileScreen/types";
import dummyAvator from "assets/dummyAvator.png";
import Image from "react-native-fast-image";
import { UserTags } from "./Afflictions";
import Tags from "./Tags";
export default function About(props: AboutProps) {
  const {
    bio = "",
    full_name = "",
    currently = "",
    currentUser = false,
    user_profile_images = {},
    referral_by = null,
    selected_tags = [],
    invited_by_toggle = false,
    affiliations_array = [],
    seen_me_at = [],
    is_update = true,
  } = props;
  const {
    image1_original = null,
    image2_original = null,
    image3_original = null,
    image4_original = null,
    image5_original = null,
    image6_original = null,
  } = user_profile_images;
  const renderItem = (item, index) => {
    return <Tags tag={item.title} key={index} />;
  };

  return (
    <View>
      <ProfileImages
        image1={image1_original}
        image2={image2_original}
        image3={image3_original}
        image4={image4_original}
        image5={image5_original}
        image6={image6_original}
      />
      <View style={styles.padding}>
        {full_name != "" && <Text style={styles.header}>{full_name}</Text>}
        {currentUser && !is_update && (
          <>
            <Text style={styles.description}>
              Your profile is a little empty!
            </Text>
            <TouchableOpacity style={styles.editButton}>
              <Text style={styles.editButtonText}>SETUP PROFILE</Text>
            </TouchableOpacity>
          </>
        )}
        {bio != "" && <Text style={[styles.description]}>{bio}</Text>}
      </View>

      {selected_tags && selected_tags?.length != 0 ? (
        <View style={styles.contentContainer}>
          {selected_tags.map((item, index) => renderItem(item, index))}
        </View>
      ) : null}
      <View style={styles.padding}>
        {invited_by_toggle && referral_by && referral_by._id != null && (
          <>
            <Text style={styles.heading}>Invited by</Text>
            <View style={styles.avatarContainer}>
              <Image
                source={
                  referral_by?.cover_image
                    ? { uri: referral_by?.cover_image }
                    : dummyAvator
                }
                style={styles.image}
              />
              <View style={styles.paddingLeft}>
                <Text style={styles.heading}>
                  {referral_by?.full_name ? referral_by?.full_name : ""}
                </Text>
                {props?.created_at && (
                  <Text style={styles.date}>
                    {moment(props?.created_at * 1000).format("DD MMMM yyyy")}
                  </Text>
                )}
              </View>
            </View>
          </>
        )}
        <Text style={styles.heading}>More about {full_name}</Text>
      </View>
      {affiliations_array && affiliations_array?.length != 0 ? (
        <>
          <Text style={styles.innerHeading}>Affiliations :</Text>
          <UserTags
            data={affiliations_array
            }
          />
        </>
      ) : null}
      {seen_me_at && seen_me_at?.length ? (
        <>
          <Text style={styles.innerHeading}>You might have seen me at :</Text>
          <UserTags
            data={seen_me_at}
          />
        </>
      ) : null}
      {currently && currently?.length ? (
        <>
          <Text style={styles.innerHeading}>Currently :</Text>
          <Text style={styles.currentlyDescription}>{currently}</Text>
        </>
      ) : null}
    </View>
  );
}

const styles = StyleSheet.create({
  header: {
    fontSize: widthPercentageToDP(10),
    fontWeight: "600",
    color: iconColor,
    textAlign: "center",
    paddingTop: 20,
  },
  description: {
    fontSize: widthPercentageToDP(3.1),
    fontWeight: "300",
    color: iconColor,
    textAlign: "center",
    paddingTop: 5,
  },
  editButton: {
    width: widthPercentageToDP(30),
    borderWidth: 1,
    alignItems: "center",
    justifyContent: "center",
    alignSelf: "center",
    marginTop: 25,
  },
  editButtonText: {
    fontSize: widthPercentageToDP(3.1),
    fontWeight: "600",
    color: iconColor,
    textAlign: "center",
    paddingVertical: 5,
  },
  padding: { paddingHorizontal: 20 },
  heading: {
    paddingTop: 20,
    fontSize: widthPercentageToDP(4),
    fontWeight: "500",
    fontFamily: appFonts.InterRegular,
    color: iconColor,
  },
  avatarContainer: {
    flexDirection: "row",
    paddingTop: 30,
    alignItems: "center",
  },
  innerHeading: {
    paddingLeft: 20,
    paddingTop: 14,
    paddingBottom: 14,
    fontSize: widthPercentageToDP(3),
    fontWeight: "600",
    fontFamily: appFonts.InterRegular,
    color: iconColor,
  },
  currentlyDescription: {
    color: iconColor,
    paddingHorizontal: 20,
    fontFamily: appFonts.InterRegular,
    fontSize: widthPercentageToDP(3.5),
    fontWeight: "400",
  },
  paddingLeft: {
    paddingLeft: 15,
  },
  contentContainer: {
    paddingLeft: 10,
    paddingTop: 20,
    flexWrap: "wrap",
    flexDirection: "row",
  },
  image: {
    width: widthPercentageToDP(15),
    height: widthPercentageToDP(15),
    borderRadius: widthPercentageToDP(7),
    borderWidth: 1,
  },
  date: {
    paddingTop: 2,
    fontSize: widthPercentageToDP(3),
    fontWeight: "400",
    fontFamily: appFonts.InterRegular,
    color: iconColor,
  },
});
