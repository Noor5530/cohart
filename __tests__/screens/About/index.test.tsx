/**
 * @format
 */


 import 'react-native';
 import React from 'react'; 
 import About from '../../../app/screens/About';
 import renderer from 'react-test-renderer';

 jest.useFakeTimers()

 
 test('renders About component correctly', () => {
   const tree = renderer.create(<About />).toJSON();
   expect(tree).toMatchSnapshot();
 });
 