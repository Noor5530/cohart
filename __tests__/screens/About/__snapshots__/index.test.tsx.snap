// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders About component correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "borderBottomColor": "#000000",
        "borderBottomWidth": 1.5,
        "flexDirection": "row",
        "height": 80,
        "justifyContent": "space-between",
        "paddingHorizontal": 20,
        "paddingTop": 30,
        "width": "100%",
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
          "padding": 10,
        }
      }
    >
      <View
        style={
          Object {
            "width": 38,
          }
        }
      >
        <RNSVGSvgView
          align="xMidYMid"
          bbHeight="14"
          bbWidth="21"
          fill="none"
          focusable={false}
          height="14"
          meetOrSlice={0}
          minX={0}
          minY={0}
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
                "borderWidth": 0,
              },
              Object {
                "flex": 0,
                "height": 14,
                "width": 21,
              },
            ]
          }
          vbHeight={14}
          vbWidth={21}
          width="21"
          xmlns="http://www.w3.org/2000/svg"
        >
          <RNSVGGroup
            fill={null}
            propList={
              Array [
                "fill",
              ]
            }
          >
            <RNSVGPath
              d="M6.64474 0.583984L0.433134 6.79559L0.167969 7.06076L0.433134 7.32592L6.64474 13.5375L7.17507 13.0072L1.66787 7.5L21 7.5V6.75L1.53939 6.75L7.17507 1.11431L6.64474 0.583984Z"
              fill={4278190080}
              propList={
                Array [
                  "fill",
                ]
              }
            />
          </RNSVGGroup>
        </RNSVGSvgView>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#000000",
              "fontSize": 20,
            },
            Object {
              "color": "#000000",
              "fontFamily": "AktivGroteskExTrial-Bold",
            },
            Object {
              "fontFamily": "AktivGroteskExTrial-Bold",
            },
          ]
        }
      >
        COHART
      </Text>
    </View>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
          "padding": 10,
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="19"
        bbWidth="16"
        fill="none"
        focusable={false}
        height="19"
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            Object {
              "flex": 0,
              "height": 19,
              "width": 16,
            },
          ]
        }
        vbHeight={19}
        vbWidth={16}
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <RNSVGGroup
          fill={null}
          propList={
            Array [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M7.58726 1.75C8.01448 1.75 8.36081 2.09633 8.36081 2.52355C8.36081 3.03874 8.66732 3.59741 9.26471 3.7987C11.462 4.53906 13.0903 6.71461 13.0903 9.32316C13.0903 9.36104 13.0899 9.39882 13.0893 9.43649C13.0881 9.50183 13.0931 9.56582 13.1039 9.62789V12.5105C13.1039 12.8394 13.1805 13.1637 13.3275 13.4579L14.1596 15.1219C14.199 15.2007 14.1417 15.2934 14.0536 15.2934H1.11992C1.03185 15.2934 0.974564 15.2007 1.01395 15.1219L1.81191 13.526C1.95899 13.2319 2.03556 12.9075 2.03556 12.5786V9.75294C2.06936 9.64853 2.08682 9.53692 2.08501 9.42124C2.0845 9.38862 2.08424 9.35593 2.08424 9.32316C2.08424 6.71462 3.71252 4.53907 5.90981 3.7987C6.5072 3.59741 6.81371 3.03874 6.81371 2.52355C6.81371 2.09633 7.16004 1.75 7.58726 1.75ZM9.58401 2.85105C9.44664 2.80476 9.36081 2.66851 9.36081 2.52355C9.36081 1.54405 8.56676 0.75 7.58726 0.75C6.60775 0.75 5.81371 1.54405 5.81371 2.52355C5.81371 2.66851 5.72787 2.80476 5.5905 2.85105C2.97586 3.73204 1.08424 6.29665 1.08424 9.32316C1.08424 9.36116 1.08454 9.39908 1.08513 9.43693C1.08535 9.45079 1.07419 9.46217 1.06033 9.46217C1.04665 9.46217 1.03556 9.47326 1.03556 9.48694V12.5786C1.03556 12.7523 0.995133 12.9235 0.91748 13.0788L0.119523 14.6747C-0.252313 15.4184 0.288463 16.2934 1.11992 16.2934H5.15932C5.40744 17.4128 6.406 18.25 7.6001 18.25C8.79419 18.25 9.79276 17.4128 10.0409 16.2934H14.0536C14.8851 16.2934 15.4258 15.4184 15.054 14.6747L14.222 13.0107C14.1443 12.8554 14.1039 12.6841 14.1039 12.5105V9.46957C14.1039 9.46549 14.1006 9.46217 14.0965 9.46217C14.0923 9.46217 14.089 9.45878 14.0891 9.45464C14.0899 9.41091 14.0903 9.36709 14.0903 9.32316C14.0903 6.29665 12.1987 3.73203 9.58401 2.85105ZM8.99864 16.2934H6.20155C6.41923 16.8532 6.96332 17.25 7.6001 17.25C8.23687 17.25 8.78097 16.8532 8.99864 16.2934Z"
            fill={4278190080}
            propList={
              Array [
                "fill",
              ]
            }
          />
        </RNSVGGroup>
      </RNSVGSvgView>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "alignSelf": "center",
        "borderBottomWidth": 2,
        "borderColor": "#000000",
        "flexDirection": "row",
        "paddingTop": 35,
        "width": "90%",
      }
    }
  >
    <Text
      numberOfLines={3}
      style={
        Object {
          "fontFamily": "Inter-Bold",
          "fontSize": undefined,
          "maxWidth": "90%",
          "textAlign": "left",
        }
      }
    >
      About
    </Text>
    <View
      style={
        Object {
          "height": "100%",
        }
      }
    >
      <View
        style={
          Object {
            "marginBottom": 4,
            "marginLeft": -25,
            "marginTop": 5,
          }
        }
      >
        <RNSVGSvgView
          align="xMidYMid"
          bbHeight="81"
          bbWidth="91"
          fill="none"
          focusable={false}
          height="81"
          meetOrSlice={0}
          minX={0}
          minY={0}
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
                "borderWidth": 0,
              },
              Object {
                "flex": 0,
                "height": 81,
                "width": 91,
              },
            ]
          }
          vbHeight={81}
          vbWidth={91}
          width="91"
          xmlns="http://www.w3.org/2000/svg"
        >
          <RNSVGGroup
            fill={null}
            propList={
              Array [
                "fill",
              ]
            }
          >
            <RNSVGPath
              d="M0.256306 80.85L45.5 0.306278L90.7437 80.85H0.256306Z"
              propList={
                Array [
                  "stroke",
                ]
              }
              stroke={4278190080}
            />
          </RNSVGGroup>
        </RNSVGSvgView>
      </View>
    </View>
  </View>
  <RCTScrollView
    contentContainerStyle={
      Object {
        "paddingHorizontal": 36,
        "paddingVertical": 35,
      }
    }
    showsVerticalScrollIndicator={false}
  >
    <View>
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="48"
        bbWidth="48"
        fill="none"
        focusable={false}
        height="48"
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            Object {
              "flex": 0,
              "height": 48,
              "width": 48,
            },
          ]
        }
        vbHeight={48}
        vbWidth={48}
        width="48"
        xmlns="http://www.w3.org/2000/svg"
      >
        <RNSVGGroup
          fill={null}
          propList={
            Array [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M24.6002 22.5478V0H23.3985V22.5495L7.45346 6.60432L6.60375 7.45403L22.5544 23.4048H0V24.6065H22.5415L6.60375 40.5444L7.45346 41.3941L23.3985 25.4489V47.998H24.6002V25.4506L40.5435 41.3941L41.3932 40.5444L25.4555 24.6065H47.9985V23.4048H25.4426L41.3932 7.45403L40.5435 6.60432L24.6002 22.5478Z"
            fill={4278190080}
            propList={
              Array [
                "fill",
              ]
            }
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <Text
        style={
          Object {
            "color": "#000000",
            "fontFamily": "Inter-Regular",
            "fontSize": 20,
          }
        }
      >
        Connecting the Creative
        

        Community
      </Text>
      <Text
        style={
          Object {
            "fontSize": 14,
            "paddingTop": 30,
          }
        }
      >
        VERSION 
         (
        )
      </Text>
      <Text
        style={
          Object {
            "color": "#000000",
            "fontFamily": "Inter-Regular",
            "fontSize": 15,
            "marginTop": 22,
          }
        }
      >
        Cohart is a female-founded platform which enables visual creators and their audiences to build authentic connections and communities.
        


        Our focus is on unlocking new ways for visual creators to monetize their passions, while building an experience for audiences to interact with these creators in a fun and engaging way.
        


        Cohart is for for everyone. We make art discovery accessible by opening windows into the worlds of creators.
      </Text>
    </View>
  </RCTScrollView>
</View>
`;
